tests/infra/network.py

```diff
@@ -147,6 +148,11 @@ def __init__(
         self.dbg_nodes = dbg_nodes
         self.perf_nodes = perf_nodes
 
+        # Requires admin privileges
+        self.partitioner = (
```

Comment:
Because this required admin privileges, should we move all partition tests under a different `ctest` label so that it is easy to exclude them when running all tests from the command line? In other words, `ctest -L e2e` should always work, but `ctest -L partitions` would require `sudo`?

Reply:
Yes, this is an excellent idea.

Reply:
Done. As per `test.yml`, we use this to run the partitions test with `sudo` in a separate step than the other non-privileged end-to-end tests.